src/axom/primal/examples/primal_introduction.cpp

```diff
@@ -157,11 +157,11 @@ PolygonType showClip()
     "{0}"
     "// axes\n"
     "draw(O -- 1.7X, arrow=Arrow3(DefaultHead2), "
-      "L=Label(\"$x$\", position=EndPoint, align=W));\n"
+    "L=Label(\"$x$\", position=EndPoint, align=W));\n"
```

Comment:
I've been leaving this one alone since we don't currently have an automatic way of regenerating the figures and I wasn't sure how sensitive asymptote is to white space changes, if at all.

@agcapps ?

Reply:
well, these are all C++ strings that get concatenated.  The whitespace changes should be fine.  They already got broken up because of a strict max-80-column formatting rule...

Reply:
Thanks @agcapps